src/components/Search.jsx

```diff
@@ -0,0 +1,9 @@
+const Search = ({ className }) => {
```

Comment:
💊 제안
input의 다른 속성도 받을 수 있게 해주시면 사용성이 더 좋아질 것 같아요!
추가로 검색 기능이 요구사항에 없는데 구현해보시면 더 좋을 것 같아요~

```suggestion
const Search = ({ className='', ...props }) => {
```

```suggestion
const Search = ({ className='', onChange }) => {
```